www/js/splash/startprefs.js

```diff
@@ -37,12 +37,16 @@ angular.module('emission.splash.startprefs', ['emission.plugin.logger',
     };
 
     var writeConsentToNative = function() {
-      return $window.cordova.plugins.BEMDataCollection.markConsented($rootScope.req_consent);
+     return $window.cordova.plugins.BEMUserCache.putRWDocument(CONSENTED_KEY, 
```

Comment:
I did this when I could not log in because of the consent but it works with the consent now.